train.py

```diff
@@ -397,23 +404,31 @@ def eval_model(global_step, writer, device, model, checkpoint_dir, ismultispeake
     # hard coded
     speaker_ids = [0, 1, 10] if ismultispeaker else [None]
     for speaker_id in speaker_ids:
-        speaker_str = "multispeaker{}".format(speaker_id) if speaker_id is not None else "single"
+        speaker_str = "multispeaker{}".format(
+            speaker_id) if speaker_id is not None else "single"
 
         for idx, text in enumerate(texts):
             signal, alignment, _, mel = synthesis.tts(
                 model_eval, text, p=0, speaker_id=speaker_id, fast=True)
             signal /= np.max(np.abs(signal))
 
             # Alignment
-            path = join(eval_output_dir, "step{:09d}_text{}_{}_alignment.png".format(
-                global_step, idx, speaker_str))
-            save_alignment(path, alignment)
-            tag = "eval_averaged_alignment_{}_{}".format(idx, speaker_str)
-            writer.add_image(tag, np.uint8(cm.viridis(np.flip(alignment, 1).T) * 255), global_step)
+            try:
+                path = join(eval_output_dir, "step{:09d}_text{}_{}_alignment.png".format(
+                    global_step, idx, speaker_str))
+                save_alignment(path, alignment)
+                tag = "eval_averaged_alignment_{}_{}".format(idx, speaker_str)
+                writer.add_image(tag, np.uint8(cm.viridis(
+                    np.flip(alignment, 1).T) * 255), global_step)
+            except Exception as e:
```

Comment:
Please keep `try: ... except` as minimal as possible. I think 

```py
try:
   writer.add_image(...)
```

is better.